notebooks/Rhythm/tabla_solo_dataset_paths.py

```diff
@@ -0,0 +1,82 @@
+# as downloaded from compmusic
+annotations_paths = [
+    '/Volumes/Shruti/data/tabla_solo_1.0/onsNoMap/ajr_10_ajr.csv',
```

Comment:
volumes/shruti? Should this stay like that?

Reply:
This was to serve as an example, until we have the example of how to use a dataset not in MIR i guess this can stay, when we have it we can remoce it